prepare-brevdata-grunnlag/pom.xml

```diff
@@ -86,6 +97,23 @@
 
 	<build>
 		<plugins>
+
```

Comment:
Plugin for å generere Request-/Respons-klasser for GraphQL endepunktet til PDL basert på schema